### ISSUE TITLE
The fine-tuning training process

### ISSUE THREAD

**Issue by xiaobingbuhuitou** [oxpig/AntiFold#1]
Hello, it's a great work. The paper mentions several mask schemes in training processes could you open-sourced training process? Thanks.😀

**Comment by tuln128** [comment]
@xiaobingbuhuitou: I have the same question, but found that the thread was closed with no further feedback. Have you been able to figure out how to acquire the codes of finetuning process?
Many thanks!
